src/app/home/HomeComponent.tsx

```diff
@@ -278,7 +278,7 @@ export default connect(
     planStatusCounts: planSelectors.getCounts(state),
     allClusters: clusterSelectors.getAllClusters(state),
     allStorage: storageSelectors.getAllStorage(state),
-    allPlans: planSelectors.getAllPlans(state),
+    allPlans: planSelectors.getPlansWithStatus(state),
```

Comment:
This seems dangerous. I don't think we would want to set a property called `allPlans` to a filtered list of plans without a status. Also it's still valid to show plans without status in the data list (dunno if this would actually change that or not).

Reply:
@eriknelson This is actually adding a status field in the selector that the ui is using to calculate status based on other values from each plan object.